template/src/components/header/index.js

```diff
@@ -13,67 +13,85 @@ import 'preact-material-components/Toolbar/style.css';
 import style from './style';
 
 export default class Header extends Component {
-	closeDrawer(){
+	closeDrawer() {
 		this.drawer.MDComponent.open = false;
 		this.state = {
 			darkThemeEnabled: false
 		};
 	}
+
+	openDrawer = () => (this.drawer.MDComponent.open = true);
+
+	openSettings = () => this.dialog.MDComponent.show();
+
+	drawerRef = drawer => (this.drawer = drawer);
+	dialogRef = dialog => (this.dialog = dialog);
+
+	linkTo = path => () => {
```

Comment:
this has to be  `linkTo = path => {...}`  right?

Reply:
linkTo returns a function that goes to the address you pass into it. I used it to make the goHome and goToMyProfile functions without repeating myself as much.

Reply:
oh my bad.....totally forgot about that part.